test/functional/cmdline_options_tester/src/Test.java

```diff
@@ -409,7 +409,9 @@ public void dumpVerboseOutput(boolean result) {
 
 	public static int getUnixPID(Process process) throws Exception {
 		Class<?> cl = process.getClass();
-		if (!cl.getName().equals("java.lang.UNIXProcess")) {
+		if (!cl.getName().equals("java.lang.UNIXProcess") 
+		|| !cl.getName().equals("java.lang.ProcessImpl")  /* intended for Fedora_Linux/RISC-V */
```

Comment:
This is going to break Linux.

Reply:
Nope, the change was verified on personal builds (including all platforms)

Reply:
If you are concerned with the potential issue,  we can add the check to ensure it only exists on RISC-V.

Reply:
When cl is UNIXProcess, `if (!(true) || !(false))` or `if (false || true)` is always true, so on Linux it will return 0. This code is only used when a timeout occurs, so a test would have to hang in order to see it no longer works on Unix.

> we can add the check to ensure it only exists on RISC-V.

this would be better

Reply:
Added as suggested above.